logs_analyzer/app/scala/src/main/scala/com/databricks/apps/logs/ApacheAccessLog.scala

```diff
@@ -0,0 +1,42 @@
+package com.databricks.apps.logs
+
+/**
+  * An entry of Apache access log.
+  */
```

Comment:
Although lots of comments like this are found in Scala-land, they don't follow the desire for reasonable terseness espoused by Scala's philosophy. You could shorten this, and eliminate the blank line before the case class:

    /** An entry of Apache access log. */